components/Cards/SchoolCard/SchoolCard.js

```diff
@@ -70,7 +70,11 @@ export default class SchoolCard extends Component {
 
     return (
       <Card className={styles.SchoolCard} hasAnimationOnHover={false} data-testid="SchoolCard">
-        {hasGiBill && <div className={styles.giBillRibbon}>GI Bill</div>}
+        {hasGiBill && (
+          <div className={styles.giBillRibbon} data-testid="GIBill">
```

Comment:
Can we be more descriptive with this? Maybe `data-testid="GI Bill Ribbon"`?

That will make my desired assertion read rather nicely.

Reply:
Completed this change

Reply:
Did you forget to push, perhaps?

Reply:
Sorry doing it now.

Reply:
got to fix lint error